app/scripts/lib/validate.js

```diff
@@ -13,6 +13,11 @@ define(function (require, exports, module) {
   // taken from the fxa-auth-server
   var HEX_STRING = /^(?:[a-fA-F0-9]{2})+$/;
 
+  // URL RegEx taken from http://blog.mattheworiordan.com/post/13174566389/url-regular-expression-for-links-with-or-without
```

Comment:
should this be moved into https://github.com/mozilla/fxa-content-server/blob/master/app/scripts/lib/url.js